src/kvstore/src/main.rs

```diff
@@ -67,6 +82,16 @@ struct GetOpt {
     hex: bool,
 }
 
+#[derive(Parser)]
+struct QueryOpt {
+    /// The key to get.
+    key: String,
+
+    /// If the key is passed as an hexadecimal string, pass this key.
+    #[clap(long)]
```

Comment:
Should be mutually exclusive.

Reply:
`hex_key` is a flag. If set to `true`, it means `key` is a hex string to decode.